code/_core/obj/item/storage/bags/_bags.dm

```diff
@@ -327,4 +327,124 @@
 /obj/item/storage/pillbottle/antihol_small/fill_inventory()
 	for(var/i=1,i<=10,i++)
 		new /obj/item/container/pill/antihol(src)
+	return ..()
+
+/obj/item/storage/bagofhoarding
+	name = "bag of hoarding"
+	desc = "we just warframe now"
+	desc_extended = "A peculiar bag that can hold unlimited amounts of specific rare currencies. Can be dyed."
+	icon = 'icons/obj/item/storage/boxes.dmi'
+	icon_state = "moneybag"
+	value = 10000
+	var/hoard = 0				// how many inside
+	var/fuck					// this one is used for the base currency roots, look further down for comments
```

Comment:
Just a minor nitpick, the variable should be somewhat straightforward without comments. It just makes it easier for me and future contributors to understand.

Reply:
FAIR ENOUGH i set it that due to the mental damage makin this gave me, should be fixed now plus i got to squash a save-bug at the same time